workers/tests/integrationtests/test_cli.py

```diff
@@ -127,6 +127,14 @@ def test_get_available_regions(self):
         self.assertEqual(result.exit_code, 0)
         self.assertTrue(geojson.loads(result.output).is_valid)
 
+    def test_get_indicator_layer_combination(self):
```

Comment:
This test fails for me. The CLI does not output a JSON or Dict like structure as the API does. Instead it just prints/echo out multiple lines of text (string).